pyiron_workflow/macro.py

```diff
@@ -317,13 +317,10 @@ def __init__(
         if len(ui_nodes) > 0:
             self._whitelist_inputs_map(*ui_nodes)
         if returned_has_channel_objects is not None:
+            if not isinstance(returned_has_channel_objects, tuple):
```

Comment:
This was not intentional (also nothing changed). I can revoke this change if you prefer.

Reply:
Maybe we lack a test where the graph creator returns only a single value?

Reply:
Ah, nvm, I see your comment applies to the change in the next few lines too. It's just a refactor. I don't have strong feelings, I find both perfectly readable and don't see an efficiency difference.